src/main/java/TaskManager.java

```diff
@@ -0,0 +1,134 @@
+public class TaskManager {
+    private Task[] taskList;
+    private int listIndex;
+
+    public TaskManager() {
+        taskList = new Task[100];
+        listIndex = 0;
+        initAlfred();
+    }
+
+    private String line = "____________________________________________________________\n";
+
+    private void initAlfred() {
+        String logo =
+                " **********************************\n" +
+                " *     _    _  __              _  *\n" +
+                " *    / \\  | |/ _|_ __ ___  __| | *\n" +
+                " *   / _ \\ | | |_| '__/ _ \\/ _` | *\n" +
+                " *  / ___ \\| |  _| | |  __/ (_| | *\n" +
+                " * /_/   \\_\\_|_| |_|  \\___|\\__,_| *\n" +
+                " **********************************\n";
+        System.out.println(
+                logo + "\n" +
+                line +
+                " Welcome back, Master Wayne.\n" +
+                " How may I be of service to you?\n" +
+                line
+        );
+    }
+
+    public void shutdownMessage() {
+        System.out.println(
+                line +
+                " Very well sir, I shall leave you to your own devices.\n" +
+                line
+        );
+    }
+
+    public void processInput(String userInput) {
+        String[] destructuredInput = userInput.split(" ");
```

Comment:
Plural form should be used on names representing a collection of objects
```suggestion
        String[] destructuredInputs = userInput.split(" ");
```

Reply:
Noted, thanks.